[RUM-6587] mention the Host header in RUM proxy guide (#25771)
DataDog/documentation: content/en/real_user_monitoring/guide/proxy-rum-data.md

1. [Build the Datadog intake URL](#build-the-datadog-intake-url).
2. Add an `X-Forwarded-For` header containing the request client IP address for accurate geoIP.

> 3. Remove the `Host` header or set it to Datadog intake URL's host (ex: `browser-intake-datadoghq.com`) (your HTTP client will probably add it automatically).

Reviewer: ```suggestion
3. Remove the `Host` header or set it to Datadog intake URL's host, such as `browser-intake-datadoghq.com`.
```
